doc/CodingStandard.md

```diff
@@ -421,6 +436,7 @@ if (NULL != myPointer) {
 * The ") {" sequence provides a clear visual indication that the condition has ended and the block has begun.
 * C order of operation rules are complicated; avoid confusion and mistakes by using parentheses to make the order explicit.
 * Combining assignment and tests violates the "Try to do only one thing on each line of code" principle.
+* When comparing a value with two other values as ((min <= value) && (value <= max)), it is clearer.
```

Comment:
This can be removed since the above suggestion includes the justification next to the example.